src/monsters/turael/CaveSlime.ts

```diff
@@ -0,0 +1,37 @@
+import LootTable from '../../structures/LootTable';
+import SimpleMonster from '../../structures/SimpleMonster';
+import { GemTable } from '../subtables/RareDropTable';
+
+export const CaveSlimeTable = new LootTable()
+	.every('Swamp tar', [1, 6])
+	.tertiary(128, 'Clue scroll (easy)', 1)
+
+	/* Weapons and Armour */
+
+	.addItem('Bronze axe', 1, 16)
+	.addItem('Iron sword', 1, 12)
+	.addItem('Bronze full helm', 1, 14)
+	.addItem('Iron kiteshield', 1, 10)
+	.addItem('Iron boots', 1, 1)
+	/*Runes */
+	.addItem('Water rune', 15, 20)
+	.addItem('Earth rune', 5, 7)
+	/*Other */
+	.addItem('Coins', 1, 30)
```

Comment:
remove all the coin drops and replace it with a table
	.add( 
		new LootTable() 
			.add('Coins', 1),
			.add('Coins', 4),
			.add('Coins', 10), 
			.add('Coins', 22), 
			.add('Coins', 46)
		x 
	)